data/repos.yml

```diff
@@ -338,6 +338,13 @@
   sentry_url: false
   private_repo: true
 
+- repo_name: govuk-graphql
+  team: "#govuk-publishing-platform"
+  alerts_team: "#govuk-publishing-platform"
```

Comment:
Could this be updated to use our automated alerts channel
```suggestion
  alerts_team: "#govuk-publishing-platform-system-alerts"
```

Reply:
Yep, Aga caught it too - fixed. Thanks!